docs/github-app.adoc

```diff
@@ -52,6 +52,13 @@ Permissions this plugin uses:
 - Pull requests: Read-only
 - Webhooks (optional) - If you want the plugin to manage webhooks for you, Read and Write
 
+You can let the app handle all events needed by this plugin.
+In that case it will not try to manage webhooks for you (and does not need webhook permissions).
+You should ask it to receive at least the following events:
+
+- Pull request
+- Push
+- Repository (additionally requires read-only Administration permission)
```

Comment:
```suggestion
You are not required to use webhooks but it strongly recommeded over alternatives such as polling. To use webhooks, you can give the your GitHub App "Read and Write" permission to Webhooks (noted above) and the plugin will configure your repositories to send the appropriate events.  Otherwise you can manually configure the app handle all events needed by this plugin, in which case the app will not need webhook permissions.

You should ask the App to receive at least the following events:

- Pull request
- Push
- Repository (automatically add  required read-only "Administration" permission)
```